site.hs

```diff
@@ -21,9 +21,18 @@ import qualified Data.Text as T
 main :: IO ()
 main = hakyll $ do
 -- statics ---------------------------------------------------------------------------------------------
-    match "assets/css/main.css" $ do
-        route   idRoute
-        compile compressCssCompiler
+    match "assets/css/tailwind.css" $ do
+        route idRoute
+        -- We concatenate a dev.css file that exists at the root of the repository so that people don't
+        -- need to have nodejs setup or working in order to get a functional development experience
+        -- "why yes this is very crimes why do you ask"
+        compile $ getResourceString >>= traverse (unixFilter "cat" ["-", "dev.css"])
```

Comment:
Won't this make it not work on Windows? I'd like Windows users to be able to contribute here.

Reply:
It _may_ work on Windows? I'm not sure.

That said, this was the ugliest/quickest possible thing I could think of. I will put a few calories into actually writing a real solution here to make sure it works on windows :)